### ISSUE TITLE
Why exclude 'playground' namespace by default ?

### ISSUE THREAD

**Issue by dregad** [LotarProject/dokuwiki-plugin-datatables#41]
I just wasted 10 minutes trying to figure out why the plugin was not working, until I realized that _playground_ namespace is [disabled by default in config ](https://github.com/LotarProject/dokuwiki-plugin-datatables/blob/419ac10/conf/default.php#L9).

This does not make sense to me, as the playground is usually where one wants to test things out...

**Comment by listerr** [comment]
Yes. I just fell into exactly the same rabbithole. Finally searched for bug reports. Thanks for this issue!
